tests/pipelines/stable_diffusion_2/test_stable_diffusion_attend_and_excite.py

```diff
@@ -34,7 +34,6 @@
 
 
 torch.backends.cuda.matmul.allow_tf32 = False
-torch.use_deterministic_algorithms(False)
```

Comment:
This will get ran once when the module is loaded and then determinism will get turned back on later